portal/templates/flask_user/second_factor.html

```diff
@@ -0,0 +1,16 @@
+{% extends "layout.html" %}
+{% block main %}
+{% from "flask_user/_macros.html" import render_field %}
+
+    <h3 class="tnth-headline">{{ _("Two Factor Athentication") }}</h3><br />
+
+    <p>{{ _("You should receive an email with a token.  Enter below for access.") }}</p>
```

Comment:
Just a little more explicit (acknowledge Mo may have more language tweaks)
```suggestion
    <p>{{ _("You should receive an email with a token. Enter the token below for access.") }}</p>
```